### PR TITLE
Broke out the static variables so that they can be used separately.

### PR DESCRIPTION
**Pull request by eitanbanks**
### Description

I want to use the individual constants outside of the list, so need them broken out.

### Checklist

- [x] Code compiles correctly
- [x] New tests covering changes and new functionality
- [x] All tests passing
- [x] Extended the README / documentation, if necessary
- [x] Is not backward compatible (breaks binary or source compatibility)

### CODE REVIEW

**Comment by lbergelson** [comment]
@eitanbanks 👍  to merge when tests pass, but can you update the commit name and comment when you merge so it's clear what constants are being split out.

File: src/main/java/htsjdk/samtools/util/IOUtil.java [hunk 1]
```diff
@@ -87,7 +87,10 @@
     public static final long FIVE_GBS = 5 * ONE_GB;
 
     /** Possible extensions for VCF files and related formats. */
-    public static final String[] VCF_EXTENSIONS = new String[] {".vcf", ".vcf.gz", ".bcf"};
+    public static final String VCF_EXTENSION = ".vcf";
+    public static final String BCF_EXTENSION = ".bcf";
+    public static final String COMPRESSED_VCF_EXTENSION = ".vcf.gz";
+    public static final String[] VCF_EXTENSIONS = new String[] {VCF_EXTENSION, COMPRESSED_VCF_EXTENSION, BCF_EXTENSION};
```
Comment (pshapiro4broad) [inline src/main/java/htsjdk/samtools/util/IOUtil.java]: You don't need `new String[]` when you're using an array literal as an initializer. You can write it like this:
```
 public static final String[] VCF_EXTENSIONS = {VCF_EXTENSION, COMPRESSED_VCF_EXTENSION, BCF_EXTENSION};
```

File: src/main/java/htsjdk/samtools/util/IOUtil.java [hunk 2]
```diff
@@ -87,7 +87,10 @@
     public static final long FIVE_GBS = 5 * ONE_GB;
 
     /** Possible extensions for VCF files and related formats. */
```
Comment (pshapiro4broad) [inline src/main/java/htsjdk/samtools/util/IOUtil.java]: This only applies to the field `VCF_EXTENSION`. Maybe it should be moved down to the line before `VCF_EXTENSIONS`. And you could add javadocs for the other new public fields.

File: src/main/java/htsjdk/samtools/util/IOUtil.java [hunk 3]
```diff
@@ -87,7 +87,10 @@
...
-    public static final String[] VCF_EXTENSIONS = new String[] {".vcf", ".vcf.gz", ".bcf"};
+    public static final String VCF_EXTENSION = ".vcf";
```
Comment (pshapiro4broad) [inline src/main/java/htsjdk/samtools/util/IOUtil.java]: Below I see fields named `XXX_FILE_EXTENSION` so the names here aren't consistent with them. I'm not sure how important consistency is in this file.

File: src/main/java/htsjdk/samtools/util/IOUtil.java (hunk 1 above)
Comment (eitanbanks) [inline src/main/java/htsjdk/samtools/util/IOUtil.java]: Bah, I was just copying what was already there...
Alright, fine.

**Pull request merged**